less/home-page.less

```diff
@@ -56,6 +72,139 @@
     .destination {
       width: 30%;
       margin-bottom: 30px;
+      // text-align: justify;
+
+      .btn {
+        width: 80%;
+        .btns(teal, 1.7rem, white);
+      }
+
+      .btn:hover {
+        background-color: #004040;
+      }
+    }
+  }
+
+  @media @mobile {
+    margin: 0 auto;
+    padding: 1.5rem;
+
+    .header-content {
+      margin: 0 auto;
+      display: flex;
+      flex-direction: column;
+      border-bottom: 1px dashed gray;
+      padding-bottom: 20px;
+
+      img {
+        text-align: center;
+        padding-bottom: 20px;
+      }
+
+      p {
+        text-align: justify;
+        font-size: 1.2rem;
+      }
+
+      h2 {
+        font-size: 2rem;
+      }
+    }
+
+    .content-section {
+      margin: 30px 0;
+      display: flex;
+      flex-direction: column;
+      flex-wrap: nowrap;
+      justify-content: space-between;
+
+      .text-content {
+        width: 100%;
+        // padding-right: 1%;
+
+        h2 {
+          font-size: 2rem;
+        }
+
+        p {
+          font-size: 1.2rem;
+        }
+      }
+
+      .img-content {
+        width: 100%;
+        // padding-left: 1%;
+
+        img {
+          border-radius: 10px;
+        }
+      }
+    }
+
+    .inverse-content {
+      display: flex;
+      flex-direction: column-reverse;
+      padding-bottom: 30px;
+      border-bottom: 1px dashed @navigation-border;
+
+      .text-content {
+        // padding-left: 1%;
+        padding-right: 0;
+      }
+
+      .img-content {
+        padding-right: 1%;
+        padding-left: 0;
+      }
+    }
+
+    .content-destination {
+      width: 100%;
+      margin: 0 auto 30px;
+
+      h2 {
+        font-size: 2rem;
+      }
+
+      p {
+        font-size: 1.2rem;
+      }
+
+      img {
+        border-radius: 10px;
+      }
+    }
+
+    .content-pick {
+      padding-top: 30px;
+      border-top: 1px dashed @navigation-border;
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+      justify-content: space-between;
+
+      h2 {
+        font-size: 2rem;
+      }
+
+      p {
+        font-size: 1.2rem;
+      }
+
+      .destination {
+        width: 80%;
+        margin-bottom: 30px;
+        // text-align: justify;
+
+        .btn {
+          width: 100%;
+          .btns(teal, 1.7rem, white);
+        }
+
+        .btn:hover {
```

Comment:
You could've nested this inside your `.btn {}` selector. It would look something like this:
```
.btn {
          width: 100%;
          .btns(teal, 1.7rem, white);
          &:hover {
               background: #004040;
            }
        }